sourcecode/apis/contentauthor/composer.json

```diff
@@ -14,45 +14,44 @@
         "ext-libxml": "*",
         "ext-pdo": "*",
         "ext-zip": "*",
-        "cerpus/cerpushelper": "^2.1",
+        "cerpus/cerpushelper": "^2.2",
         "cerpus/edlib-resource-kit": "^0.7.x-dev",
         "cerpus/edlib-resource-kit-laravel": "^0.7.x-dev",
-        "cerpus/imageservice-client": "^2.1",
-        "cerpus/questionbank-client": "^2.3.1",
-        "ckeditor/ckeditor": "^4.5",
-        "daniel-km/simple-iso-639-3": "^0.1.1",
-        "doctrine/dbal": "^2.5",
-        "embed/embed": "^3.2",
-        "firebase/php-jwt": "^6.3",
-        "guzzlehttp/guzzle": "^7.4",
+        "cerpus/imageservice-client": "^2.2.1",
+        "cerpus/questionbank-client": "^2.3.3",
+        "ckeditor/ckeditor": "^4.24",
```

Comment:
Latest (and last) open source version is 4.22.1. Versions >= 4.23.0 requires a license key available to customers who have purchased Extended Support Model Package

Reply:
Added a conflict to ensure it isn't updated again.